api/Element.json

```
@@ -2458,10 +2461,11 @@
               "notes": "This API was previously available on the<code>Node</code> API."
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": "46",
+              "notes": "This API was previously available on the<code>Node</code> API."
```

Since this line is being modifed anyway, mind adding the missing space here?

```suggestion
              "notes": "This API was previously available on the <code>Node</code> API."
```